jjb/ci-management/ci-jobs.yaml

```diff
@@ -10,13 +10,17 @@
     project-name: ci-management
     build-node: centos7-builder-1c-1g
     jjb-version: 3.5.0
+    jenkins-silos: production
 
 - project:
     name: github-ci-jobs
 
     jobs:
-      - '{project-name}-github-ci-jobs':
-          jenkins-silos: production
+      - github-jenkins-cfg-merge
+      - github-jenkins-cfg-verify
+      - github-jjb-deploy-job
```

Comment:
Since there is no sandbox jenkins, there is no reason to have the jjb-deploy-job definition as that is just a way to push an open ci-management change into a sandbox for testing.